src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -1070,6 +1075,22 @@ private UsernamePasswordAuthenticationToken loginAndSetUserData(
             user.setFullName(fullName);
         }
 
+        // Set avatar if possible
+        try {
+            String avatarUrl = determineStringField(avatarFieldExpr, idToken, userInfo);
+            if (avatarUrl != null) {
+                LOGGER.finest("Avatar url is: " + avatarUrl);
```

Comment:
This does string concatenation and then in most cases throws the result away.
Better to use parameterized logging or a supplier.

Additionally, would it be useful to know which user this was for?

(untested) 
```suggestion
                LOGGER.finest(() -> {"Avatar for user `" + user.getId() + "` url is: " + avatarUrl });
```